pkg/monitors/http/http.go

```diff
@@ -264,6 +264,7 @@ func (m *Monitor) getTLSStats(site *url.URL, logger *logrus.Entry) (dps []*datap
 
 	dimensions := map[string]string{
 		"server_name": host,
+		"sni_server_name": serverName,
```

Comment:
The linter is saying some issue on this line.  See if you can figure out what -- it isn't obvious to me, maybe tabs vs spaces issue.

Reply:
@keitwb 
```
max@mde-claranet % gofmt -s pkg/monitors/http/http.go | sponge pkg/monitors/http/http.go
max@mde-claranet % git diff
diff --git a/pkg/monitors/http/http.go b/pkg/monitors/http/http.go
index dad8b665..fd8e9335 100644
--- a/pkg/monitors/http/http.go
+++ b/pkg/monitors/http/http.go
@@ -263,7 +263,7 @@ func (m *Monitor) getTLSStats(site *url.URL, logger *logrus.Entry) (dps []*datap
        }

        dimensions := map[string]string{
-               "server_name": host,
+               "server_name":     host,
                "sni_server_name": serverName,
        }

max@mde-claranet % git push -f perso http_sni
```
I fixed and force-push.